client/src/pages/Profile/ProfileSetting.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react';
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import { NavLink, useHistory } from 'react-router-dom';
+import { CircularProgress, Grid, Box, ListItem, List } from '@mui/material';
+import { Navbar } from '../../components/Navbar/Navbar';
+import useStyles from './useStyles';
+import { Link } from 'react-router-dom';
+import { BrowserRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
+import ProfilePhoto from '../../components/Profile/ProfilePhoto';
+import Availabilty from '../../components/Profile/Availabilty';
+import EditProfile from '../../components/Profile/EditProfile';
+
+export default function ProfileSetting(): JSX.Element {
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+  const history = useHistory();
+  const classes = useStyles();
+  useEffect(() => {
```

Comment:
We don't need to initialize the socket connection here, you can remove this hook